subsys/fs/shell.c

```diff
@@ -39,7 +39,6 @@ FS_LITTLEFS_DECLARE_DEFAULT_CONFIG(lfs_data);
 static struct fs_mount_t littlefs_mnt = {
 	.type = FS_LITTLEFS,
 	.fs_data = &lfs_data,
-	.storage_dev = (void *)STORAGE_PARTITION_ID,
```

Comment:
(note) looks like, you can make smarter solution as it was done here https://github.com/xen-troops/zephyr/pull/19

Reply:
Thanks for suggestion, switched to linked approach